src/main/java/com/esri/samples/portal/oauth/OAuthChallengeHandler.java

```diff
@@ -0,0 +1,117 @@
+/*

```

Comment:
Might be able to store this class in its own package and reference it from there since this and `Create and Save Sample` both use it.